ssn/index.html

```diff
@@ -1508,9 +1628,39 @@ <h5><a href="#SOSAActuation">sosa:Actuation</a></h5>
           <p><strong>an OWL Class</strong></p>
 
           <em property="rdfs:label">Actuation</em> -
-          <span property="rdfs:comment skos:definition">An <a href="#SOSAActuation"><code>Actuation</code></a> carries
-            out an (Actuation) <a href="#SOSAProcedure"><code>Procedure</code></a> to change the state of the world
-            using an <a href="#SOSAActuator"><code>Actuator</code></a>.</span> <br>
+          <span property="skos:definition">An <a href="#SOSAActuation"><code>Actuation</code></a> carries out an
+            (Actuation) <a href="#SOSAProcedure"><code>Procedure</code></a> to change the state of something in the
+            world using an <a href="#SOSAActuator"><code>Actuator</code></a>.
+            <br>
+            <br>
+            In this context,
+            <br>
+            - <a href="#SOSAhasResult"><code>hasResult</code></a> shall indicate the intended value of the <a
+              href="#SOSAActuatableProperty"><code>ActuatableProperty</code></a> at the completion of the <a
+              href="#SOSAActuation"><code>Actuation</code></a>
+            <br>
+            - <a href="#SOSAresultTime"><code>resultTIme</code></a> shall indicate the time that the <a
+              href="#SOSAActuation"><code>Actuation</code></a> was initiated
```

Comment:
I know this is what we agreed on during the call but now that I read it, I find the statement about `resultTime` really counter intuitive!  (there is a typo on `resultTIme` vs `resultTime` but the way)

Our use of `hasResult` was ok with good explanations because the provided values are indeed the intended result. But `resultTime` really sounds like the time when the result of the actuation is attained, don't you think?

What about defining the `resultTime` as the time the intended result has been achieved.

And `phenomenonTime` as the time that the actuation start (or even the time period during which the actuation occurs since it can be a time period)?

The rationale being that the phenomenon in the case of the actuation is whatever the actuator does in the real world to change the property.